`test_accessions_fields` does not assert anything

https://github.com/DataBiosphere/azul/blob/7fb38a696a70b3ce5b76287786c0feb13ede1423/test/indexer/test_indexer.py#L1416

Because the index name is actually "projects" (plural), this block never executes and no assertions are ever performed. If the pluralization issue is fixed, and the assertions execute as intended, the test fails because the orderings of the lists are different.

```
AssertionError: Lists differ: [{'namespace': 'array_express', 'accession': 'E-AAAA-00'}[224 chars]00'}] != [{'namespace': 'insdc_project', 'accession': 'SRP000000'}[224 chars]00'}]

First differing element 0:
{'namespace': 'array_express', 'accession': 'E-AAAA-00'}
{'namespace': 'insdc_project', 'accession': 'SRP000000'}

- [{'accession': 'E-AAAA-00', 'namespace': 'array_express'},
+ [{'accession': 'SRP000000', 'namespace': 'insdc_project'},
+  {'accession': 'SRP000001', 'namespace': 'insdc_project'},
   {'accession': 'GSE00000', 'namespace': 'geo_series'},
+  {'accession': 'E-AAAA-00', 'namespace': 'array_express'},
-  {'accession': 'SRP000000', 'namespace': 'insdc_project'},
-  {'accession': 'SRP000001', 'namespace': 'insdc_project'},
   {'accession': 'PRJNA000000', 'namespace': 'insdc_study'}]
```

- [ ] Security design review completed; the Resolution of this issue does **not** …
  - [ ] … affect authentication; for example:
    - OAuth 2.0 with the application (API or Swagger UI)
    - Authentication of developers with Google Cloud APIs
    - Authentication of developers with AWS APIs
    - Authentication with a GitLab instance in the system
    - Password and 2FA authentication with GitHub
    - API access token authentication with GitHub
    - Authentication with 
  - [ ] … affect the permissions of internal users like access to
    - Cloud resources on AWS and GCP
    - GitLab repositories, projects and groups, administration
    - an EC2 instance via SSH
    - GitHub issues, pull requests, commits, commit statuses, wikis, repositories, organizations
  - [ ] … affect the permissions of external users like access to
    - TDR snapshots
  - [ ] … affect permissions of service or bot accounts
    - Cloud resources on AWS and GCP
  - [ ] … affect audit logging in the system, like
    - adding, removing or changing a log message that represents an auditable event
    - changing the routing of log messages through the system
  - [ ] … affect monitoring of the system
  - [ ] … introduce a new software dependency like
    - Python packages on PYPI
    - Command-line utilities
    - Docker images
    - Terraform providers
  - [ ] … add an interface that exposes sensitive or confidential data at the security boundary
  - [ ] … affect the encryption of data at rest
  - [ ] … require persistence of sensitive or confidential data that might require encryption at rest
  - [ ] … require unencrypted transmission of data within the security boundary
  - [ ] … affect the network security layer; for example by 
    - modifying, adding or removing firewall rules
    - modifying, adding or removing security groups
    - changing or adding a port a service, proxy or load balancer listens on
- [ ] Documentation on any unchecked boxes is provided in comments below

--- a/test/indexer/test_indexer.py
+++ b/test/indexer/test_indexer.py
@@ -1407,9 +1407,9 @@ class TestHCAIndexer(DCP1TestCase, IndexerTestCase):
             contents = hit['_source']['contents']
             project = one(contents['projects'])
             accessions_by_namespace = {
-                'array_express': ['E-AAAA-00'],
-                'geo_series': ['GSE00000'],
                 'insdc_project': ['SRP000000', 'SRP000001'],
+                'geo_series': ['GSE00000'],
+                'array_express': ['E-AAAA-00'],
                 'insdc_study': ['PRJNA000000']
             }
             entity_type, aggregate = self._parse_index_name(hit)

--- a/test/indexer/test_indexer.py
+++ b/test/indexer/test_indexer.py
@@ -1413,7 +1413,7 @@ class TestHCAIndexer(DCP1TestCase, IndexerTestCase):
                 'insdc_study': ['PRJNA000000']
             }
             entity_type, aggregate = self._parse_index_name(hit)
-            if entity_type == 'project':
+            if entity_type == 'projects':
                 expected_accessions = [
                     {'namespace': namespace, 'accession': accession}
                     for namespace, accessions in accessions_by_namespace.items()

